enhancements/installer/vsphere-ipi-multiple-datacenter-clusters.md

```diff
@@ -0,0 +1,472 @@
+---
+title: multiple-datacenters-cluster-vsphere-ipi
+authors:
+  - "@jcpowermac"
+reviewers:
+  - "@rvanderp3"
+  - "@bostrt"
+  - ""
+approvers:
+  - ""
+creation-date: 2021-09-21
+last-updated: 2021-10-21
+status: implementable
+see-also:
+  - "/enhancements/"
+replaces:
+  -
+superseded-by:
+  -
+---
+
+# Support Multiple vSphere vCenter Datacenters and Clusters in IPI
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The goal of this enhancement is to provide the ability to install in a
+vSphere enviornment with multiple datacenters and clusters.
+
+This will be an opionated design, the vCenter datacenter will always be a `region`
+and a vCenter cluster will always be a `zone`.
+
+## Motivation
+
+Users of OpenShift would like the ability to deploy
+within multiple datacenters and clusters to increase
+reliablity. Customers would also like to take advantage
+of the concept of regions and zoning that this type of deployment would
+offer.
+
+### Goals
+
+- Support regions and zones in vSphere using multiple datacenters (region) and
+clusters (zone)
+- Support installation into multiple datacenters and multiple clusters
+
+### Non-Goals
+
+- Support for VMware vSphere in-tree or out-of-tree storage provider
+- Support multiple subnets
+- Support multiple vcenters
+
+Note: The platform spec will be modified to support this at a future date.
+Only a single item in `vcenters` will be supported for the initial release.
+
+## Proposal
+
+Modification of the installer to support the provisioning of masters in defined
+datacenters and clusters.
+
+### installer
+
+#### Platform spec
+
+The platform spec needs to be modified to support our initial goals of multiple
+datacenters (regions) and clusters (zones) and vcenters.
+
+We are adding two additonal parameters to the Platform struct:
+
+- `VCenters`
+- `Regions`
+
+`VCenters` will contain the connection details for each individual vcenter, the
+existing platform spec vcenter parameters will be depreciated but _not_ removed
```

Comment:
Given that, are there any concerns about backward compatibility to be described as part of the enhancement? E.g. on `openshift-install destroy`.